benchmarks/01_trivial/07_loop_unroll_varying-cpu.ispc

```diff
@@ -0,0 +1,412 @@
+#include "../../ispcrt/ispcrt.isph"
+
+
+// Macros
+
+#define UNROLL_VARYING_SIG_CPU(T_FOR, T_ISPC, UNROLL_FACTOR)              \
+    export void unroll_varying_cpu_##T_FOR##_##T_ISPC##_##UNROLL_FACTOR(  \
+        uniform T_ISPC src[],                                             \
+        uniform T_ISPC dst[],                                             \
+        uniform uint64 num_elems                                          \
+    )
+
+#define UNROLL_VARYING_FOREACH_BODY   \
+    foreach (i = 1 ... num_elems) {   \
+        dst[i] = src[i] + src[i - 1]; \
```

Comment:
As a note - alternatively we can have computations with loop carried dependency to observe good gains thanks to unrolling. But current implementation is good enough.